content/en/shader-editor/window-layout/graph-editor.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Graph Editor
+template: shader-editor-page.tmpl.html
+position: 4
+---
+
+The Graph Editor displays the currently selected graph. This is where nodes are created, arranged, connected and deleted.
+
+![Graph Editor][1]
+
+| How To | |
+|---|---|
+| Create a node | Drag an entry from the [Nodes Pane][2] and drop it on the Graph Editor work area.<br>Drag a sub-graph from the [Assets Pane][3] onto the Graph Editor work area. |
+| Select a node | Left click the node. |
+| Delete a node | Select the node and either press delete key or use the node context menu 'Delete' option. |
+| Move a node | Left click and drag the node. |
+| Modify a node | Select the node and modify its properties in the [Inspector Pane][4]. |
+| Connect two nodes | Left click the output port of the source node, drag and drop on the input port of the target node. |
```

Comment:
Some of these would be great with GIFs, especially 'Connect two nodes'

Reply:
Or maybe just a general video/GIF of the operations?

Reply:
yup, followup pr?

Reply:
Yep